models/DataObject/ClassDefinition.php

```diff
@@ -551,7 +551,7 @@ public function generateClassFiles($generateDefinitionFile = true)
 
         if (is_array($this->getFieldDefinitions()) && count($this->getFieldDefinitions())) {
             foreach ($this->getFieldDefinitions() as $key => $def) {
-                if (!$def instanceof DataObject\ClassDefinition\Data\ReverseObjectRelation && !$def instanceof DataObject\ClassDefinition\Data\CalculatedValue
+                if (!$def instanceof DataObject\ClassDefinition\Data\CalculatedValue
```

Comment:
This can be reverted, no need to include a property that doesn't get used at all 😉

Reply:
Done